catalogs/sources/gtfs/schedule/ro-brasov-regia-autonoma-de-transport-brasov-gtfs-2143.json

```diff
@@ -19,5 +19,10 @@
         "direct_download": "https://github.com/szjozsef/osm2gtfs/raw/refs/heads/master/output/gtfs/ro-ratbv.zip",
```

Comment:
Don't redirect, the new feed looks unofficial. I'd add the second one but mark it as unofficial data